### PR TITLE
fix prow

### PR DESCRIPTION
**Pull request by philbrookes**
# Description

### CODE REVIEW

File: pkg/config/threescale.go [hunk 1]
```diff
@@ -101,3 +102,30 @@ func (t *ThreeScale) Validate() error {
...
+			"apicastStage":    1,
+			"backendListener": 1,
+			"backendWorker":   1,
+			"backendCron":     1,
+			"zyncApp":         1,
+			"zyncQue":         1,
+		}
+	}
+	return map[string]int64{
+		"systemApp":       2,
```
Comment (maleck13) [inline pkg/config/threescale.go:121]: change to 3

File: pkg/config/threescale.go [hunk 2]
```diff
@@ -101,3 +102,30 @@ func (t *ThreeScale) Validate() error {
...
+		"systemSidekiq":   2,
```
Comment (maleck13) [inline pkg/config/threescale.go:122]: 3

File: pkg/config/threescale.go [hunk 3]
```diff
@@ -101,3 +102,30 @@ func (t *ThreeScale) Validate() error {
...
+		"apicastProd":     6,
+		"apicastStage":    3,
+		"backendListener": 5,
+		"backendWorker":   4,
+		"backendCron":     2,
```
Comment (maleck13) [inline pkg/config/threescale.go:127]: 1

File: pkg/config/threescale.go [hunk 4]
```diff
@@ -101,3 +102,30 @@ func (t *ThreeScale) Validate() error {
...
+		"zyncApp":         2,
```
Comment (maleck13) [inline pkg/config/threescale.go:128]: 3

File: pkg/config/threescale.go [hunk 5]
```diff
@@ -101,3 +102,30 @@ func (t *ThreeScale) Validate() error {
...
+		"zyncQue":         2,
```
Comment (maleck13) [inline pkg/config/threescale.go:129]: 3

**Comment by philbrookes** [comment]
/test rhoam-e2e 
failed due to PROW issues

File: test/common/scale_up_replicas_rhsso.go [hunk 6]
```diff
@@ -14,20 +15,36 @@ import (
...
-func TestReplicasInRHSSOAndUserSSO(t *testing.T, ctx *TestingContext) {
-	checkScalingOfKeycloakReplicas(t, ctx, rhssoName, GetPrefixedNamespace("rhsso"), numberOfRhssoReplicas)
-	checkScalingOfKeycloakReplicas(t, ctx, userSSOName, GetPrefixedNamespace("user-sso"), numberOfUserRhssoReplicas)
+func TestReplicasInRHSSO(t *testing.T, ctx *TestingContext) {
+	inst, err := GetRHMI(ctx.Client, true)
+	if err != nil {
+		t.Fatalf("failed to get RHMI instance %v", err)
+	}
+	if resources.RunningInProw(inst) {
+		checkScalingOfKeycloakReplicas(t, ctx, userSSOName, GetPrefixedNamespace("user-sso"), 1)
```
Comment (KevFan) [inline test/common/scale_up_replicas_rhsso.go:32]:
```suggestion
		checkScalingOfKeycloakReplicas(t, ctx, rhssoName, GetPrefixedNamespace("rhsso"), 1)
```
Should this not be RHSSO instead ? :thinking:

**Comment by philbrookes** [comment]
/test e2e 
PROW failure ... again!

**Pull request merged**